readme.md

```diff
@@ -249,4 +262,4 @@ hue.setRandomColors(["abc123", "def123"])
 [npm]: https://www.npmjs.com/package/@bradgarropy/hue-sdk
 [issues]: https://github.com/bradgarropy/hue-sdk/issues
 [twitter]: https://twitter.com/bradgarropy
-[ama]: https://bradgarropy.com/ama
+[ama]: https://bradgarropy.com/ama
```

Comment:
I wasn't sure on this one @bradgarropy ? It looks like you removed it? Feel free to change it as you see fit! I left the PR open to allow changes from owners

Reply:
Yes, this should definitely be there!